icu4c/source/i18n/unicode/numsys.h

```diff
@@ -106,9 +106,9 @@ class U_I18N_API NumberingSystem : public UObject {
 
     /**
      * Return a StringEnumeration over all the names of numbering systems known to ICU.
+     * The numbering system names will be in alphabetical order.
```

Comment:
Not sure if perhaps a silly comment/remark, but would it be worth saying that they are in alphabetical (invariant) order? (Just to clarify that it isn't based on the default locale or anything like that.)